aspnetcore/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/Views/Shared/_Layout.cshtml

```diff
@@ -29,7 +29,7 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a asp-area="" asp-controller="Movies" asp-action="Index" class="navbar-brand">MvcMovie</a>
+                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">MvcMovie</a>
```

Comment:
I guess it doesn't matter for the download - except I have the following in the instructions
![image](https://cloud.githubusercontent.com/assets/3605364/22902846/9cbe476c-f1da-11e6-98a7-fe1b8caa7454.png)